frontend/public/components/storage/attach-storage.tsx

```diff
@@ -117,6 +125,27 @@ export const AttachStorageForm: React.FC<AttachStorageFormProps> = (props) => {
     validateMountPaths(event.currentTarget.value);
   };
 
+  const getDevicePaths = (podTemplate: any): string[] => {
+    const containers: ContainerSpec[] = _.get(podTemplate, 'spec.containers', []);
+    return containers.reduce((acc: string[], container: ContainerSpec) => {
+      if (!isContainerSelected(container)) {
+        return acc;
+      }
+      const devicePaths: string[] = _.map(container.volumeDevices, 'devicePath');
+      return acc.concat(devicePaths);
+    }, []);
+  };
+
+  const validateDevicePath = (path: string) => {
+    const existingDevicePaths = getDevicePaths(obj.spec.template);
+    const err = existingDevicePaths.includes(path) ? 'Device path is already in use.' : '';
+    setError(err);
```

Comment:
Could this clear a different error that might have been previously set?

Reply:
But sometimes you call `setError` with `''` as the value. It will clear any previous error in that case.

Reply:
Ahh.... i got your point now.  The changes should be suffice now